specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json

```diff
@@ -118,6 +118,58 @@
         }
       }
     },
+    "/providers/Microsoft.Subscription/subscriptionOperations/{operationId}": {
+      "get": {
+        "description": "Get the status of the pending Microsoft.Subscription API operations.",
+        "operationId": "SubscriptionOperation_Get",
+        "x-ms-examples": {
+          "getPendingSubscriptionOperations": {
+            "$ref": "./examples/getSubscriptionOperation.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
> "x-ms-long-running-operation": true, [](http://example.com/codeflow?start=7&length=37)

Why is this GET LRO? Get should not be async and it should just return 200, as the linter error also marked

Reply:
This is one of the existing api, support this version earlier present in 2019-10-01preview. Recently we added this version in manifest to support this api, that' why coz trigger s360 item for missing operation in the swagger.

For code ref :- https://github.com/abhaygill338/azure-rest-api-specs/blob/sub-operations-missing-2021-10-01/specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

Reply:
The preview version had GET sync, 200 and 202, and since it's sync, is 202 used at all? And in this PR it is async + 200 + 202 on GET, async on GET is not supported in any client I believe. You should only use sync + 200 on GET. This will also fix your lint error

Reply:
And the purpose of this API is to get the operation status, the status can be completed or pending, but that is not the signature of this API, it is just the state of the result, the API itself should just be a straightforward GET to get the state as the result. Your LIST call actually has the right pattern, see ["/providers/Microsoft.Subscription/operations" ](https://github.com/Azure/azure-rest-api-specs/blob/4936e00f30d12f9c8d7f3e490336d47f4df1449b/specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json#L174)

Reply:
Sure, GetSubscriptionOperations the manifest updated with the version https://armmanifest.visualstudio.com/One/_git/Manifest/commit/d575e742371c1e1e7741d840cc6b87383a24e02e?refName=refs/heads/master&path=/MICROSOFT.SUBSCRIPTION/PROD/MANIFEST.JSON&_a=compare, pls have a look at this commit.

Reply:
This API should be sync right? My understanding was you guys are just returning the wrong respond code, but it is a sync API like in the preview version?

Reply:
Please make the corresponding update, and add suppression to the respond code error, we can review the suppression